src/variorum/ARM/power_features.h

```diff
@@ -17,19 +17,19 @@ void init_arm(void);
 
 void shutdown_arm(void);
 
-int get_power_data(int verbose, FILE *output);
+int arm_cpu_get_power_data(int verbose, FILE *output);
```

Comment:
Changing `"power_features.h"` back to `<power_features.h>` didn't cause the `multiple definition of ...` issue at linking since the rest of the fix enables function name resolution. But it brought back the following:

`...
warning: implicit declaration of function ‘init_arm’; did you mean ‘initNVML’? [-Wimplicit-function-declaration]
     init_arm();
...`

It's not a critical error and doesn't affect the functionality in the existing code base. In this case, should we skip this change to using `" "` to include `power_features.h` or keep it?